app/backend/approaches/approach.py

```diff
@@ -96,7 +96,7 @@ def __init__(
         self.openai_host = openai_host
 
     def build_filter(self, overrides: dict[str, Any], auth_claims: dict[str, Any]) -> Optional[str]:
-        exclude_category = overrides.get("exclude_category") or None
+        exclude_category = overrides.get("exclude_category")
```

Comment:
right .get defaults to None as the returned value if it's not present. I think I helped copy paste this everywher sorry about that